README.md

```diff
@@ -226,6 +226,10 @@ This section provides information about the notebooks and the order that they sh
     * [analysis/performance_analysis/merged_performance_analysis.ipynb](./analysis/performance_analysis/merged_performance_analysis.ipynb)
       * Analysis on Variation Normalizer performance on CIViC, MOA, and ClinVar
 
+## Results
```

Comment:
`Running Notebooks in Visual Studio Code (VS Code)` is not a sub section for Results. 

```suggestion
### Notebook Results
```